tests/system/providers/microsoft/psrp/example_psrp.py

```diff
@@ -0,0 +1,79 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+This is an example dag for using the PsrpOperator.
+"""
+from __future__ import annotations
+
+# --------------------------------------------------------------------------------
+# Load The Dependencies
+# --------------------------------------------------------------------------------
+from datetime import datetime, timedelta
+
+from airflow import DAG
+
+# --------------------------------------------------------------------------------
+# The purpose of this is to give you a sample of a real world example DAG!
+# --------------------------------------------------------------------------------
+
+
+try:
+    from airflow.operators.empty import EmptyOperator
+except ModuleNotFoundError:
+    from airflow.operators.dummy import DummyOperator as EmptyOperator  # type: ignore
```

Comment:
I'm not sure that we should fallback to `DummyOperator` because `EmptyOperator` introduced into the airflow 2.3, and next release of provider would have requirement min Airflow 2.5

Tis may not apply to examples/system tests so let's wait and see what the other says

Reply:
I'll keep this in mind for the final iteration before moving it out of draft. I keep getting this error now:
```
/opt/airflow/docs/apache-airflow-providers-microsoft-psrp/_api/tests/system/providers/microsoft/psrp/index.rst: WARNING: document isn't included in any toctree
```

And this happens after I've added the following to docs/apache-airflow-providers-microsoft-psrp:


`Example DAGs <https://github.com/apache/airflow/tree/providers-microsoft-psrp/|version|/tests/system/providers/microsoft/winrm>`


And added the example_psrp to 
`tests/system/providers/microsoft/psrp/`

Reply:
You could add your changes into the PR, this would be way easier to find what is going wrong

Reply:
Just pushed it, thanks.